document.go

```diff
@@ -18,6 +18,12 @@ type Document struct {
 	// But DisplayedCursorPosition returns 4 because '日' and '本' are double width characters.
 	cursorPosition int
 	lastKey        Key
+
+	// lines_ is the cache for function `Lines`.
+	lines_ []string
+
+	// lineStartIndexes_ is the cach function `lineStartIndexes`.
+	lineStartIndexes_ []int
```

Comment:
In Go there are no need at the `_` postfixes.

Reply:
Fixed.